.gitignore

```diff
@@ -14,3 +14,4 @@
 /tags
 /tmp/*
 /spec/example_app/tmp/*
+.idea
```

Comment:
Never knew there was global .gitignore, thanks for the knowledge!

I'll revert it